src/middlewares/secretsManager.js

```diff
@@ -111,3 +111,13 @@ const getSecretsManagerInstance = awsSdkOptions => {
   const { SecretsManager } = require('aws-sdk')
   return new SecretsManager(awsSdkOptions)
 }
+
+function safeParse (secretString) {
+  try {
+    return JSON.parse(secretString || '{}')
+  } catch (err) {
+    console.info('plain string')
```

Comment:
how about `return secretString` here?

Reply:
Agreed, I would leave the `catch` block empty and return the original unparsed string.

As an alternative, which would add a little bit more complexity, we could expose the functionality to handle errors, by giving the user the possibility to inject a function in the config that will be called here with the given error and the given original string.